src/stale-prs.ts

```diff
@@ -340,6 +342,30 @@ export class StalePrFinder {
 
     return ret;
   }
+
+  /**
+   * Returns true if the PR has been reviewed or commented on by a core member.
+   */
+  private async memberReviewed(pull_number: number): Promise<Boolean> {
+    
+    const comments = (await this.client.paginate(this.client.rest.issues.listComments, { ...this.repo, issue_number: pull_number, }));
+    
+    // Comments by team members
+    // Filtering out instances where submitted_at is empty
+    const memberComments = comments.filter(r => r.author_association === 'MEMBER').filter(r => r.submitted_at);
+
+    if (!(memberComments.length > 0)) {
```

Comment:
Turn this into an early return, instead of a large nested `if`:

```ts
if (memberComments.length > 0) {
  return true;
}

// Else check for another source of truth
```

In general, try to avoid long `if` blocks if you can; the code will be easier to follow that way.